packages/react-art/src/ReactARTHostConfig.js

```diff
@@ -11,6 +11,7 @@ import Mode from 'art/modes/current';
 import {TYPES, EVENT_TYPES, childrenAsString} from './ReactARTInternals';
 
 import {DefaultEventPriority} from 'react-reconciler/src/ReactEventPriorities';
+import {Instance} from 'chalk';
```

Comment:
This is not where `Instance` should be imported from. It should come from `ReactNativeHostConfig` although it doesn't matter much since this renderer doesn't even get Flow checked.

Reply:
So should i import it correctly or close this one?

Reply:
Probably not worth updating this renderer, to be honest.